NineChronicles.Headless/Controllers/GraphQLController.cs

```diff
@@ -261,5 +282,16 @@ private void NotifyAction(ActionBase.ActionEvaluation<ActionBase> eval)
                 }
             }
         }
+
+        // FIXME: remove this method with DI.
+        private IActionResult AuthenticateLocalPolicy()
```

Comment:
Is it okay to return only `Unauthorized` and `null`?

Reply:
At least, it seems okay in this method to me. Can there be a problem?

Reply:
`null` means "authorized"?

Reply:
No, it doesn't. I refactored it like 7615507. How about these changes?

Reply:
It make more sense to me :)